Feat/cp 32 switch language v2

windforce852 opened the pull request:
feat: add i18n switch language

- added i18n dir to app dir
- added en.json and zh-cn.json to i18n dir
- added I18nWrapper to i18n dir
- nested I18nWrapper into provider.tsx

language can be switched from AppSetting page
i18n can affect sidebarContent and AppSetting page now
further implement to other page will be execute after code review

Chris-Wang reviewed (changes requested):
needs:
- squash commits
- fix failed test

windforce852 commented:
done squash commits
fixed failed test included eslint error, and unit test for i18n

AppSetting page dropdown gloss fix applied here but further UI improvement will and should be done in CP36 (another choice is to combine both ticket into one PR)

SeijaHuang committed:
Feat add update account setting page
Add internationalization
-	Implemented internationalization using next-intl library
-	Implemented language switch between English and Chinese
Add theme switch
-	Implemented theme switch between light and dark in accounting setting page
Refactor account setting page
-	Modularized account setting page

Resolve CP-22

NEED TO BE DONE
-	Unit test for all pages implemented internationalization

NEED TO BE DONE
-	Unit test for all pages implemented internationalization
